lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart

```diff
@@ -26,103 +29,126 @@ class ExpansionContactListTile extends StatelessWidget {
+          key: contact.matrixId != null ? Key(contact.matrixId!) : null,
+          future: contact.status == ContactStatus.active
+              ? getProfile(context)
+              : null,
+          builder: (context, snapshot) {
+            return Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                IgnorePointer(
+                  child: Avatar(
+                    mxContent: snapshot.data?.avatarUrl,
+                    name: contact.displayName,
+                  ),
+                ),
+                const SizedBox(
+                  width: 12.0,
                 ),
-              ),
-              const SizedBox(
-                width: 12.0,
-              ),
-              Expanded(
-                child: Column(
-                  mainAxisSize: MainAxisSize.min,
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    IntrinsicWidth(
-                      child: Row(
-                        children: [
-                          Expanded(
-                            child: Row(
-                              children: [
-                                Flexible(
-                                  child: BuildDisplayName(
-                                    profileDisplayName:
-                                        snapshot.data?.displayName,
-                                    contactDisplayName: contact.displayName,
-                                    highlightKeyword: highlightKeyword,
-                                    style: Theme.of(context)
-                                        .textTheme
-                                        .titleMedium
-                                        ?.copyWith(
-                                          color: Theme.of(context)
-                                              .colorScheme
-                                              .onSurface,
-                                        ),
+                Expanded(
+                  child: Column(
+                    mainAxisSize: MainAxisSize.min,
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      IntrinsicWidth(
+                        child: Row(
+                          children: [
+                            Expanded(
+                              child: Row(
+                                children: [
+                                  Flexible(
+                                    child: BuildDisplayName(
+                                      profileDisplayName:
+                                          snapshot.data?.displayName,
+                                      contactDisplayName: contact.displayName,
+                                      highlightKeyword: highlightKeyword,
+                                      style: LinagoraTextStyle.material()
+                                          .bodyMedium2
```

Comment:
you use it exactly, but when I run in web app, the weight of this text is not precise

<img width="304" alt="image" src="https://github.com/user-attachments/assets/c3d83a85-9bc9-4db2-9de2-c4d46ce9aad8">

Reply:
and color is not true too.

Reply:
fixed